src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -12,31 +12,89 @@
 namespace Symfony\Bundle\WebProfilerBundle\Twig;
 
 use Symfony\Component\HttpKernel\DataCollector\Util\ValueExporter;
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Dumper\HtmlDumper;
 
 /**
  * Twig extension for the profiler.
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class WebProfilerExtension extends \Twig_Extension
+class WebProfilerExtension extends \Twig_Extension_Profiler
```

Comment:
this is required, and there is no complict: the two inserted nodes won't be the same because they get the extension name as first parameter, which make them different. see comment below for the reason why we need to track enter/leave here.